src/program/program.spec.ts

```diff
@@ -47,10 +54,13 @@ describe("Plan services", () => {
 		resolver = new ProgramResolver(service);
 	});
 
-	let testingModuleID: string;
-	let testingAssignmentID: string;
+	let testingModuleID;
+	let testingAssignmentID;
 	let testingAssignmentResultID: string;
 	let testingCourseID: string;
+	let createNewContent;
```

Comment:
Create new content variable has a type of `any`, which is why you weren't able to catch logical errors further down the file